unity-renderer/Assets/DCLPlugins/ECS7/ECS7Plugin.cs

```diff
@@ -68,8 +69,8 @@ public ECS7Plugin()
                 new WrappedComponentPool<IWrappedComponent<PBEngineInfo>>(MAX_EXPECTED_SCENES, () => new ProtobufWrappedComponent<PBEngineInfo>(new PBEngineInfo())),
                 new WrappedComponentPool<IWrappedComponent<PBUiCanvasInformation>>(MAX_EXPECTED_SCENES, () => new ProtobufWrappedComponent<PBUiCanvasInformation>(new PBUiCanvasInformation())),
                 new WrappedComponentPool<IWrappedComponent<PBPointerEventsResult>>(MAX_EXPECTED_SCENES * 10, () => new ProtobufWrappedComponent<PBPointerEventsResult>(new PBPointerEventsResult())),
-                new WrappedComponentPool<IWrappedComponent<PBTweenState>>(MAX_EXPECTED_SCENES * 10, () => new ProtobufWrappedComponent<PBTweenState>(new PBTweenState()))
-            );
+                new WrappedComponentPool<IWrappedComponent<PBTweenState>>(MAX_EXPECTED_SCENES * 10, () => new ProtobufWrappedComponent<PBTweenState>(new PBTweenState())),
+                new WrappedComponentPool<IWrappedComponent<PBAudioSource>>(MAX_EXPECTED_SCENES * 10, () => new ProtobufWrappedComponent<PBAudioSource>(new PBAudioSource())));
```

Comment:
Maybe it's out of the scope and not the best time for it, but can we take the opportunity to change the magic number 10 by something with a meaningful name? I'm new so maybe I'm not aware of the team's conventions yet, but something like MAX_SCENES_MODIFIER might make sense?

Also, just for personal learning, why some components use the MAX_EXPECTED_SCENES without the multiplication by 10? Maybe it's a oversight and they also need it and because of that, maybe we would just need to increase MAX_EXPECTED_SCENES?

Reply:
@AlejandroAlvarezMelucciDCL I looked at this but each component has arbitrary rules as to how big to pre-allocate the pools, for example transform only has *2, so this appears to be have been determined through some sort of experiementation.

@pravusjif may be able to offer insight into what this means!

Reply:
Thanks for the explanation!

Reply:
Uff those hardcoded values are from before I joined the SDK team so I'm not really sure but I would also trust them to have been defined based on some manual testing, most likely. I'd leave them as they are on unity-renderer at lesat, we won't be using those values in the new explorer.